features/openhab-demo-resources/src/main/resources/things/demo.things

```diff
@@ -1,3 +1,4 @@
 yahooweather:weather:berlin [ location=638242 ]
-astro:sun:home  [ geolocation="52.5200066,13.4049540", interval=60 ]
+astro:sun:local  [ geolocation="52.5200066,13.4049540", interval=60 ]
```

Comment:
You should not rename "home" to "local", since this won't be the local information for anybody outside Berlin.
You could rename it to "berlin" though, if you wish.